features/step_definitions/logging.rb

```diff
@@ -535,10 +556,23 @@
         | n            | #{project_name}          |
       })
       step %Q/the step should succeed/
-      if project_name != "openshift-logging"
-        step %Q/I create pipelinesecret named "fluentdserver" with sharedkey "fluentdserver"/
+      # set configmap and create pipeline secret in openshift-logging project
+      case auth_type
+      when "mTLS_share"
+        step %Q/I create pipelinesecret with auth type mTLS_share/
+        configmap_file = "#{file_dir}/fluentd/secure/cm-mtls-share.yaml"
+      when "mTLS"
+        step %Q/I create pipelinesecret with auth type mTLS/
+        configmap_file = "#{file_dir}/fluentd/secure/cm-mtls.yaml"
+      when "server_auth"
+        step %Q/I create pipelinesecret with auth type server_auth/
+        configmap_file = "#{file_dir}/fluentd/secure/cm-serverauth.yaml"
+      when "server_auth_share"
+        step %Q/I create pipelinesecret with auth type server_auth_share/
+        configmap_file = "#{file_dir}/fluentd/secure/cm-serverauth-share.yaml"
+      else
+        raise "Unrecognized auth type: #{auth_type}"
```

Comment:
```
      case auth_type
      when "mTLS_share"
        configmap_file = "#{file_dir}/fluentd/secure/cm-mtls-share.yaml"
      when "mTLS"
        configmap_file = "#{file_dir}/fluentd/secure/cm-mtls.yaml"
      when "server_auth"
        configmap_file = "#{file_dir}/fluentd/secure/cm-serverauth.yaml"
      when "server_auth_share"
        configmap_file = "#{file_dir}/fluentd/secure/cm-serverauth-share.yaml"
      else
        raise "Unrecognized auth type: #{auth_type}"
      end
      step %Q/I create pipelinesecret with auth type #{auth_type}/
```

Reply:
Updated, could you please take a look again?